packages/synapse-react-client/src/components/EntityHeaderTable/EntityHeaderTable.tsx

```diff
@@ -229,7 +231,10 @@ export const EntityHeaderTable = (props: EntityHeaderTableProps) => {
     const newDataEntityIds = new Set()
     newData.map(entityHeader => newDataEntityIds.add(entityHeader.id))
     const missingRefs = refsInState.filter(
-      ref => !newDataEntityIds.has(ref.targetId),
+      ref =>
+        !newDataEntityIds.has(
+          ref.targetId.startsWith('syn') ? ref.targetId : `syn${ref.targetId}`,
```

Comment:
Can you extract this to a utility function called something like `normalizeSynPrefix`? You could put it in `EntityTypeUtils.ts` unless you can think of a better place